flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why? Isn't Java supposed to be platform independent?

Reply:
Yes, but in releases there isn't any aarch64 jar binary.
There's the '-any' binary, but per discussion in the now deleted issue that binary is different.

Reply:
maybe using the source to build from instead then?

Reply:
Seems like the amd64 jar contains some platform specific libraries in it while the `any` has none, so these are not equivalent.

Reply:
building from source remain the preferred method.